src/Microsoft.Extensions.Primitives/Microsoft.Extensions.Primitives.csproj

```diff
@@ -12,6 +12,10 @@ Microsoft.Extensions.Primitives.StringSegment</Description>
     <PackageTags>primitives</PackageTags>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
We should do this globally /cc @natemcmaster

Reply:
This has come up a few times lately and the answer I got from @natemcmaster is that we haven't done this because VSCode doesn't have it yet.

It also looks like Omnisharp (the library) added it 5 days ago https://github.com/OmniSharp/omnisharp-roslyn/releases but Omnisharp (the VS Code extension) hasn't yet https://github.com/OmniSharp/omnisharp-vscode/releases

Reply:
@DustinCampbell release a new omnisharp 😄

Reply:
We'll make it our default once we have better tooling support. cref https://github.com/aspnet/BuildTools/issues/566

Reply:
It's in now.